indico/modules/events/abstracts/templates/abstract.html

```diff
@@ -14,9 +14,20 @@
             {{ render_abstract_reviews(abstract) }}
             <div class="i-timeline-separator"></div>
         {% endif %}
-        {% if abstract.can_judge(session.user) %}
+
+        {% set can_review = abstract.can_review(session.user) %}
+        {% set can_judge = abstract.can_judge(session.user, check_state=true) %}
+
+        {% if can_review or can_judge %}
```

Comment:
We should also check if `abstract.state.name == 'submitted'`, otherwise we'll still get the connector with no other section afterwards.